### ISSUE TITLE
Adding new customer with already existing phone number throws invalid phone number error message

### ISSUE THREAD

**Issue by tomascherian** [tomascherian/pe#5]
**Steps to reproduce**:
* `addcustomer p/98765432 n/Adam p/ayne a/address`

![image.png](https://raw.githubusercontent.com/tomascherian/pe/main/files/a6184417-8baf-45d9-a5b9-2a974dc794e5.png)

**Expected Output**: Already existing phone number error message is thrown

**Actual Output**: Invalid phone error message is thrown

**Comment by nus-se-script** [comment]
# Team's Response
As stated in the UG, only the last occurrence of the parameter will be taken. In this case, `p/ayne` was taken to be the phone number field. 
Since `ayne` is an invalid phone number, the error message is thrown. 

![Screenshot 2022-04-18 at 3.54.04 PM.png](https://raw.githubusercontent.com/nus-cs2103-AY2122S2/pe/master/files/28ca3fb4-910a-4daf-8251-93e517597522.png)


# Items for the Tester to Verify
## :question: Issue response

Team chose [`response.Rejected`]







- [x] I disagree

**Reason for disagreement:** Thank you for the response.

I could made this bug report more clear.

The bug reported is an error message mismatch. The use of of symbols in name throws a invalid name error message. The correct error message should say the phone number already exists.

Granted `p/ayne` would make the command hard to parse, but use of other symbols like `- + #` also cause this bug. 
e.g. `addcustomer p/88993823 n/Adam-payne a/address` also causes an error message mismatch.

-------------------
